src/frontend/sharedComponents/Thumbnail/Thumbnail.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {FC} from 'react';
+import {Photo} from '../../contexts/PhotoPromiseContext/types';
+import {
+  ActivityIndicator,
+  Image,
+  ImageErrorEventData,
+  NativeSyntheticEvent,
+  StyleProp,
+  StyleSheet,
+  TouchableOpacity,
+  ViewStyle,
+} from 'react-native';
+import * as Progress from 'react-native-progress';
+import {AlertIcon} from '../icons';
+import debug from 'debug';
+import {LIGHT_GREY} from '../../lib/styles';
+
+const log = debug('Thumbnail');
+
+type ThumbnailProps = {
+  photo?: Partial<Photo>;
+  onPress: () => any;
+  style?: StyleProp<ViewStyle>;
+  size?: number;
+};
+
+export const Thumbnail: FC<ThumbnailProps> = props => {
```

Comment:
Do you think it makes sense to rename this to `PhotoThumbnail`, given that we now have `AudioThumbnail` and will eventually add `VideoThumbnail`?

Reply:
good point, thanks